src/httpation_uri.erl

```diff
@@ -203,11 +191,16 @@ consume_port(_, _, _) ->
     {error, port}.
 
 
+consume_path("", _Parts, "", URI) ->
+    {ok, URI#uri{path = ["/"]}};
 consume_path(Acc, Parts, "", URI) ->
     Part = lists:reverse(Acc),
     NewParts = [Part | Parts],
     Path = lists:reverse(NewParts),
     {ok, URI#uri{path = Path}};
+consume_path(_Acc, Parts, [$/ | [$/ | Rest]], URI) ->
```

Comment:
I have fixed that (it was not parsing properly multiples paths). 
That guard tried to properly parse `netloc////path`